src/processor/operator/ddl/drop.cpp

```diff
@@ -15,6 +15,7 @@ bool isValidEntry(parser::DropInfo& dropInfo, main::ClientContext* context) {
     case common::DropType::SEQUENCE: {
         validEntry = context->getCatalog()->containsSequence(context->getTx(), dropInfo.name);
     } break;
+        // TODO(Ziyi): If the table has indexes, we should drop those indexes as well.
```

Comment:
Maybe remove this TODO here but only put it into an issue that keeps track of the TODOs.